pennylane_qiskit/qiskit_device2.py

```diff
@@ -219,6 +218,10 @@ class QiskitDevice2(Device):
         "Hadamard",
         "Hermitian",
         "Projector",
+        "Prod",
+        "Sum",
+        "LinearCombination",
+        "SProd",
```

Comment:
I didn't add Hamiltonian or SparseHamiltonian because well
for Hamiltonian, this just gets casted to LinearCombination from what I can tell so it's kind of extraneous
for SparseHamiltonian, it seems like it's a bit of a tricky case to handle and... not worth it effort to reward ratio wise. Could be nice for the backlog :shrug:

Reply:
Might be nice to leave a `#TODO` there.